tests/Http/CurlRequestHandlerTest.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace YotiTest\Http;
+
+use Yoti\YotiClient;
+use Yoti\Http\CurlRequestHandler;
+use Yoti\Http\RequestSigner;
+use YotiTest\TestCase;
+use Yoti\Http\Payload;
+use Yoti\Entity\Country;
+use Yoti\Entity\AmlAddress;
+use Yoti\Entity\AmlProfile;
+
+class CurlRequestHandlerTest extends TestCase
+{
+    public $pem;
+    /**
+     * @var CurlRequestHandler
+     */
+    public $requestHandler;
+    /**
+     * @var Payload
+     */
+    public $payload;
+
+    public $requestSigner;
+
+    public $postMethod = CurlRequestHandler::METHOD_POST;
+
+    public function setup()
+    {
+        $endpoint = '/aml-check';
+        $amlAddress = new AmlAddress(new Country('GBR'));
+        $amlProfile = new AmlProfile('Edward Richard George', 'Heath', $amlAddress);
+        $this->payload = new Payload($amlProfile->getData());
+        $this->postMethod = CurlRequestHandler::METHOD_POST;
+
+        $this->pem = file_get_contents(PEM_FILE);
+
+        $this->requestSigner = new RequestSigner(
+            $this->requestHandler,
+            $endpoint,
+            'POST'
+        );
+
+        $this->requestHandler = new CurlRequestHandler(
+            YotiClient::DEFAULT_CONNECT_API,
+            $this->pem,
+            SDK_ID,
+            'PHP'
+        );
+    }
+
+    public function testGetPem()
+    {
+        $this->assertEquals($this->pem, $this->requestHandler->getPem());
+    }
+
+    public function testGetSdkId()
+    {
+        $this->assertEquals(SDK_ID, $this->requestHandler->getSdkId());
+    }
+
+    public function testInvalidHttpMethod()
+    {
+        $this->expectException('\Yoti\Exception\RequestException');
+        $this->requestHandler->sendRequest(
+            '/aml-check',
+            'InvalidHttpMethod',
+            $this->payload
+        );
+    }
+
+    public function testSendRequestExists()
+    {
+        $this->assertTrue(method_exists($this->requestHandler, 'sendRequest'));
```

Comment:
Not atm, but that can be added.

Reply:
The reason I was asking is because I wasn't sure about whether this test is actually necessary.
In Java I wouldn't because the compiler would tell that something is wrong otherwise.
This is why I'm not sure how it works in PHP :-)

Reply:
If PHP doesn't blow up automatically when the class is loaded, then it makes sense.

Reply:
Yes we are using composer here which follows [PSR4](https://www.php-fig.org/psr/psr-4/#1-overview), which means the classes will be automatically loaded when you refer to them by their namespace including their dependencies as set in the composer.json.